.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+### Pull Request Checklist
+
+* [ ] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog). The entry should:
```

Comment:
we might want to consider copying CONTRIBUTING.md rather than linking to the synapse one (there's a bunch of stuff there which doesn't directly apply here).

this is fine for now though.

Reply:
Yeah agreed, I just wanted to have _something_ asap so people start adding changelog files now. But yeah we should get Sydent its own CONTRIBUTING.md at some point.